client/src/ts/dial.ts

```diff
@@ -0,0 +1,187 @@
+var svgns = "http://www.w3.org/2000/svg";
```

Comment:
We should figure out how to strip the svgns dependencies throughout the project (SVG files and here).